<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RevertForm</class>
 <widget class="QDialog" name="RevertForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Revert entries</string>
  </property>
  <layout class="QVBoxLayout">
   <item>
    <widget class="QLabel" name="m_headLine">
     <property name="text">
      <string>Really revert these entries to pristine state?</string>
     </property>
     <property name="wordWrap">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QListWidget" name="m_ItemsList"/>
   </item>
   <item>
    <widget class="DepthSelector" name="m_DepthSelect" native="true">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>20</height>
      </size>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>DepthSelector</class>
   <extends>QWidget</extends>
   <header>ksvnwidgets/depthselector.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <includes>
  <include location="local">ksvnwidgets/depthselector.h</include>
 </includes>
 <resources/>
 <connections/>
</ui>
